internal/services/containerapps/container_app_resource_test.go

```diff
@@ -641,6 +641,17 @@ resource "azurerm_container_app" "test" {
       }
     }
 
+    init_container {
+      name   = "init-cont-%[2]d"
+      image  = "jackofallops/azure-containerapps-python-acctest:v0.0.1"
+      cpu    = 0.25
+      memory = "0.5Gi"
+      volume_mounts {
+        name = azurerm_container_app_environment_storage.test.name
+        path = "/tmp/testdata"
+      }
+    }
```

Comment:
If this block should be `ForceNew` as per the docs, this will break the `TestAccContainerAppResource_completeUpdate` test as it would force a destroy and recreation invalidating the result. Can we check the behaviour of that block and update accordingly?

Reply:
This block should not be `ForceNew`. I updated the docs with 9122e03.